source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -1116,19 +1178,25 @@ void Graph::buildUiBaseGraph(const std::vector<mx::NodeGraphPtr>& nodeGraphs, co
     // creating uiNodes for the nodegraph
     for (mx::NodeGraphPtr nodeGraph : nodeGraphs)
     {
+        if (!includeElement(nodeGraph))
```

Comment:
Generalize filtering so that all include files are caught regardless of whether from stdlib, or other library, or loaded documents (including those added by the renderer). Also consistency with save filtering.